openassessment/workflow/models.py

```diff
@@ -660,12 +677,12 @@ def update(self, submission_uuid, assessment_requirements):
         assessment_finished = getattr(self.api(), 'assessment_is_finished', default_finished)
```

Comment:
Why are these values only sometimes available?  Or put another way, why do we need to do a `getattr()` dance on them?